README.md

```diff
@@ -14,3 +14,35 @@ yarn add ssh:git@github.com:storypark/vue-cli-plugin-vuetify-preset-storypark.gi
 yarn add ssh:git@github.com:storypark/vue-cli-plugin-vuetify-preset-storypark.git
 vue invoke vue-cli-plugin-vuetify-preset-storypark
 ```
+
+## Development
+
+To be able to work on presets on your local machine.
+
+1. Pull the repo down to your machine
+    - a good place to have it would be the same folder that the `storyjar` repo is in
+
+2. In your terminal, make sure you are in the `storyjar` repo and in the `education` folder
+
+3. Run `yarn add ~<path to folder>/vue-cli-plugin-vuetify-preset-storypark`
+
+   e.g. `yarn add ~libby/parkstory/vue-cli-plugin-vuetify-preset-storypark`
+
+4. This will update the `package.json` to use the local preset:
+
+    - `"vue-cli-plugin-vuetify-preset-storypark": "/Users/libby/parkstory/vue-cli-plugin-vuetify-preset-storypark",`
+
+    - The `yarn.lock` file will also be updated.
+
+      **DO NOT commit this changed files.**
+
+5. Make the changes you need to in both the `vue-cli-plugin-vuetify-preset-storypark` code and the `storyjar/education` code.
+    - commit changes
+    - BUT not the changes to the `package.json` or `yarn.lock` in the `storyjar` repo
+
+6. Get the changes for the preset reviewed and merged first
+
+<br>
+
+*To add:*
+- next steps for getting the preset changes used in `storyjar`
```

Comment:
I haven't done this next part, could someone add what needs to happen and I will add it in. Thanks

Reply:
You could add an extra note about how you need to re yarn add each time you make a change, then reset your server. And could then note about the shortcut to just look under node_modules/vue-cli-plugin-vuetify-preset-storypark and make changes there, then you only need to reset the server and can miss the first step of running yarn add again

Next steps would be to post your PR up for review to the preset, get it agreed upon by the team (I think we should have a channel for any preset changes so we can all agree on them perhaps?)

Once merged, then run `yarn add https://github.com/storypark/vue-cli-plugin-vuetify-preset-storypark` which will update the lock in your PR, which you should then commit to your PR, and get it on to staging, so the live preset change can be reflected in your changes on staging, which you should then get through design review, how does that sound?

Reply:
Thanks @timothyjohn2015 . will add that stuff in. 

What do you reckon about adding this info to https://github.com/storypark/storyjar/blob/master/education/README.md#single-page-app

and then just saying in this readme to look at that readme? (😂 )

yeah I feel like that might be better since this is a public repo....

Reply:
Cool, will make a new PR in storyjar and amend this one. ta